judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiPageContainerHelper.java

```diff
@@ -109,6 +110,23 @@ public static String simpleActionDefinitionName(ActionDefinition actionDefinitio
         return relationName + (!relationName.isEmpty() ? firstToUpper(suffix) : suffix) + "Action";
     }
 
+    private static String getCallOperationName(ActionDefinition actionDefinition) {
+        OperationType operationType = null;
+        int index = 2;
+        if (actionDefinition instanceof CallOperationActionDefinition callOperationActionDefinition) {
+            operationType = callOperationActionDefinition.getOperation();
+        } else if (actionDefinition instanceof BulkCallOperationActionDefinition bulkCallOperationActionDefinition) {
+            operationType = bulkCallOperationActionDefinition.getOperation();
+        }
+        String actionDefinitionName = actionDefinition.getName();
+        String fallbackName = operationType.getName();
```

Comment:
If none of the conditions met, there will be a NPE on line 122

Reply:
This is only intended to be used for operation buttons. Other solution would be to throw an explicit exception which doesn't add much.